OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -56,36 +182,36 @@ static bool ValidateSlotCommand(S server, Connection conn, Session.Client client
 			{
 				if (!server.LobbyInfo.Slots.ContainsKey(arg))
 				{
-					Log.Write("server", "Invalid slot: {0}", arg);
+					Log.Write("server", $"Invalid slot: {arg}");
 					return false;
 				}
 
 				if (requiresHost && !client.IsAdmin)
 				{
-					server.SendOrderTo(conn, "Message", "Only the host can do that.");
+					server.SendLocalizedMessageTo(conn, RequiresHost);
 					return false;
 				}
 
 				return true;
 			}
 		}
 
-		public static bool ValidateCommand(S server, Connection conn, Session.Client client, string cmd)
+		public static bool ValidateCommand(S server, Connection conn, Session.Client client, string command)
 		{
 			lock (server.LobbyInfo)
 			{
 				// Kick command is always valid for the host
-				if (cmd.StartsWith("kick "))
+				if (command.StartsWith("kick "))
 					return true;
 
 				if (server.State == ServerState.GameStarted)
 				{
-					server.SendOrderTo(conn, "Message", $"Cannot change state when game started. ({cmd})");
+					server.SendLocalizedMessageTo(conn, StateUnchangedGameStarted, Translation.Arguments(nameof(command), command));
```

Comment:
`nameof` that should be changed here too.

Check all callers of `Translation.Arguments` for this issue.

Reply:
Removed it everywhere.